dist/preview release/what's new.md

```diff
@@ -2,351 +2,353 @@
 
 ## Major updates
 
-- Infinite Morph Targets: When supported (WebGL2+) you are no more limited to 4 morph targets per mesh ([Deltakosh](https://github.com/deltakosh))
-- Added Animation Curve Editor to allow easy animation creation and update ([Deltakosh](https://github.com/deltakosh))
-- Added support for ConditionalBlock for NodeMaterial ([Deltakosh](https://github.com/deltakosh))
-- Improved performance when using the shadow / cascacaded shadow generator ([Popov72](https://github.com/Popov72))
-- Add support for up to 6 uv sets in the standard, PBR and node materials ([Popov72](https://github.com/Popov72))
-- Added GUI Editor to easily build GUI controls ([msDestiny14](https://github.com/msDestiny14))
-- Added support for Order Independent Transparency on simple scenes. `scene.useOrderIndependentTransparency = true` now makes transparent meshes shade correctly when stacked onto each other. ([CraigFeldspar](https://github.com/CraigFeldspar))
+-   Infinite Morph Targets: When supported (WebGL2+) you are no more limited to 4 morph targets per mesh ([Deltakosh](https://github.com/deltakosh))
```

Comment:
I know the .md formatter tends to add those spaces, but we prefer not to have them. If possible, please revert this file and add your change only.

I am working on fixing the .md formatting in the repository so that this won't happen again.

Reply:
Sure!